packages/edit-site/src/components/header-edit-mode/index.js

```diff
@@ -121,7 +121,13 @@ export default function HeaderEditMode() {
 		>
 			{ hasDefaultEditorCanvasView && (
 				<motion.div
-					className="edit-site-header-edit-mode__start"
+					className={ classnames(
+						'edit-site-header-edit-mode__start',
+						{
+							'is-opened':
```

Comment:
Instead of adding a class here, maybe it's worth refactoring this to to add one or two classes to the `edit-site-header-edit-mode` that can describe the whole state of the header? 

One issue to watch ouf for here is that `showTopToolbar` is about if the toolbar _can_ render or not,  not necessarily if it will. There are sometimes where the block toolbar could be rendered but won't, such as when editing a standalone navigation from the site editor.

Reply:
I'm just using the same existing conditions that `is-collapsed` applies.

Reply:
> Instead of adding a class here, maybe it's worth refactoring this to to add one or two classes to the edit-site-header-edit-mode that can describe the whole state of the header?

Perhaps so, as each start, center, end is affected by the toolbar.

Reply:
Modified the pr as such.